inspirehep/utils/record.py

```diff
@@ -164,3 +164,13 @@ def is_complete(publication_info):
                 if is_complete(publication_info):
                     return False
     return had_at_least_one_journal_title
+
+
+def get_source(record):
+    """Return the first available source of a record."""
+    return get_value(record, 'acquisition_source.source')
+
+
+def get_control_number(record):
+    """Return the record ID."""
+    return get_value(record, 'control_number')
```

Comment:
There's no point in using `get_value` when you are just doing record access... Just do `record['control_number']` or `record.get('control_number')`.